lib/node-asset-manifest-generator.js

```diff
@@ -0,0 +1,48 @@
+var Plugin = require('broccoli-caching-writer');
+var walk = require('walk-sync');
+var path = require('path');
+var fs = require('fs-extra');
+
+/**
+ * A Broccoli plugin to generate a module to be used in Node for resolving the
+ * asset manifest. Primary use case is for FastBoot like environments.
+ *
+ * @class NodeAssetManifestGenerator
+ * @extends BroccoliCachingWriter
+ */
+function NodeAssetManifestGenerator(inputTrees, options) {
+  options = options || {};
+
+  this.appName = options.appName;
+
+  Plugin.call(this, [ inputTrees ], {
+    annotation: options.annotation
+  });
+}
+
+NodeAssetManifestGenerator.prototype = Object.create(Plugin.prototype);
+NodeAssetManifestGenerator.prototype.constructor = NodeAssetManifestGenerator;
+
+/**
+ * Generates an asset manifest module on build from the passed in
+ * asset-manifest.json file.
+ */
+NodeAssetManifestGenerator.prototype.build = function() {
+  var inputPath = this.inputPaths[0];
+  var assetManifestPath = path.join(inputPath, 'asset-manifest.json');
+  var assetManifest = fs.readJsonSync(assetManifestPath);
```

Comment:
Is this guaranteed to exist? (AKA: could this throw?)

https://github.com/trentmwillis/ember-asset-loader/blob/master/lib/asset-manifest-generator.js#L62-L66